packages/story-editor/src/components/canvas/frameElement.js

```diff
@@ -87,7 +97,17 @@ const EmptyFrame = styled.div`
 
 const NOOP = () => {};
 
+const FRAME_ELEMENT_MESSAGE = sprintf(
+  /* translators: 1: Ctrl Key 2: Alt Key */
+  __('To enter floating menu, press %1$s+%2$s+p.', 'web-stories'),
```

Comment:
💭 When chromevox is reading this out, it reads out the `+` symbol too. I think the `+` symbols could be removed

```suggestion
  __('To enter floating menu, press %1$s %2$s p.', 'web-stories'),
```